va/meson.build

```diff
@@ -55,30 +55,43 @@ libva_headers_priv = [
   'va_trace.h',
 ]
 
-libva_sym = 'libva.syms'
-libva_sym_path = '@0@/@1@'.format(meson.current_source_dir(), libva_sym)
+if host_machine.system() == 'windows'
+  libva = static_library(
```

Comment:
Why are we creating a static library on Windows where all other platforms use a shared one?
AFAICT the project aims to provide some API/ABI guarantees and using a static library bypasses all of those.

Unless there is a very compelling reason, I would stick with a shared_library on Windows.